content/_includes/birds.macros.njk

```diff
@@ -40,13 +40,16 @@ params:
     ) }}
   {% else %}
     {% for name in names %}
-      {% set tween = ' & ' | mdInline if (loop.length == 2) else ',' %}
+      {{ ' & ' if loop.last and (loop.length > 2) }}
       {{ author(
         name=name,
         collections=collections,
-        link=link
+        link=link,
+        fullname=false
       ) }}
-      {{- tween | safe if not loop.last }}
+      {%- if not loop.last -%}
+        {{ ' & ' if loop.length == 2 else ', ' }}
+      {%- endif -%}
```

Comment:
@mirisuzanne The `mdInline | safe` tags don't seem to be needed here 🤷

Reply:
@mirisuzanne I think this is back to you to merge if [my commit](https://github.com/oddbird/oddleventy/pull/143/commits/2e5b30abf374ca05dcd4e46bed306ed0a5a2da79) looks okay.